glustercli/cmd/volume.go

```diff
@@ -245,15 +245,15 @@ func volumeInfoDisplay(vol api.VolumeGetResp) {
 	fmt.Println("Volume Name:", vol.Name)
 	fmt.Println("Type:", vol.Type)
 	fmt.Println("Volume ID:", vol.ID)
-	fmt.Println("State:", vol.State)
+	fmt.Println("Status:", vol.State)
```

Comment:
I prefer "state" over "status" as it sounds more natural:

`The volume xyz is in started state` as against `The volume xyz is in started status`

But I understand that this change is driven by backwards compatibility.

Reply:
Ok, I will revert this back to `State`

Reply:
Done